strawberry/extensions/mask_errors.py

```diff
@@ -0,0 +1,54 @@
+from typing import Sequence, Type
+
+from graphql.error import GraphQLError
+
+from strawberry.extensions import Extension
+
+
+class MaskErrors(Extension):
+    visible_errors: Sequence[Type[Exception]]
+    error_message: str
+
+    def __init__(
+        self,
+        visible_errors: Sequence[Type[Exception]],
```

Comment:
What about an `exclude_errors` ? 
Sometimes you want to exclude only a few errors. Would be easier if we just pass an "exclude" instead of mentioning all the other possible errors that can happen... 
what do you think?

Reply:
I think passing an allow list is better than passing an exclude list because your exclude list is potentially unknowable and constantly expanding because any number of exceptions can be thrown. This way a new exception won't be shown to the client which I think is the right default.

I could change this API to accept a `show_show_error` predicate function instead? That way the developer could implement an allow list or an exclude list. It's a more complex API but it's more flexible.

Reply:
@estyxx I've changed the API to use a `should_mask_error` function instead. I think thats more flexible. Let me know what you think.

Reply:
@patrick91 can we do a pre-release of this, please?